.github/actions/inductor-xpu-e2e-test/action.yml

```diff
@@ -96,35 +80,54 @@ runs:
+                bash inductor_xpu_test.sh ${suite} ${dt} ${mode} ${scenario} xpu 2 static 8 2 &
+                bash inductor_xpu_test.sh ${suite} ${dt} ${mode} ${scenario} xpu 3 static 8 3 &
+                bash inductor_xpu_test.sh ${suite} ${dt} ${mode} ${scenario} xpu 4 static 8 4 &
+                bash inductor_xpu_test.sh ${suite} ${dt} ${mode} ${scenario} xpu 5 static 8 5 &
+                bash inductor_xpu_test.sh ${suite} ${dt} ${mode} ${scenario} xpu 6 static 8 6 &
+                bash inductor_xpu_test.sh ${suite} ${dt} ${mode} ${scenario} xpu 7 static 8 7 &
+                wait
+                LOG_DIR="inductor_log/${suite}/${dt}"
+                LOG_NAME=inductor_${suite}_${dt}_${mode}_xpu_${scenario}_all.log
+                rm -f ${LOG_DIR}/${LOG_NAME}
+                find ${LOG_DIR}/ -name "inductor_${suite}_${dt}_${mode}_xpu_${scenario}_card*.log" |xargs cat >> ${LOG_DIR}/${LOG_NAME} 2>&1
+                echo -e "============ Summary for ${suite} ${dt} ${mode} ${scenario} ============" | tee -a ${LOG_DIR}/${suite}_${dt}_${mode}_${scenario}_e2e_summary.log
+                awk -i inplace '!seen[$0]++' ${LOG_DIR}/inductor_${suite}_${dt}_${mode}_xpu_${scenario}.csv
+                csv_lines=$(cat ${LOG_DIR}/inductor_${suite}_${dt}_${mode}_xpu_${scenario}.csv | wc -l)
+                num_total=$(echo "$csv_lines - 1" |bc)
+                num_passed=$(grep -c "pass" ${LOG_DIR}/inductor_${suite}_${dt}_${mode}_xpu_${scenario}.csv)
+                num_failed=$(echo "$num_total - $num_passed" |bc)
+                pass_rate=`awk 'BEGIN{printf "%.2f%%\n",('$num_passed'/'$num_total')*100}'`
+                echo "num_total: $num_total" | tee -a ${LOG_DIR}/${suite}_${dt}_${mode}_${scenario}_e2e_summary.log
+                echo "num_passed: $num_passed" | tee -a ${LOG_DIR}/${suite}_${dt}_${mode}_${scenario}_e2e_summary.log
+                echo "num_failed: $num_failed" | tee -a ${LOG_DIR}/${suite}_${dt}_${mode}_${scenario}_e2e_summary.log
+                echo "pass_rate: $pass_rate" | tee -a ${LOG_DIR}/${suite}_${dt}_${mode}_${scenario}_e2e_summary.log
+              done
+            done
+          done
+        done
         cd ${{ github.workspace }} && cp -r ../pytorch/inductor_log .
+
+    - name: Upload Inductor XPU E2E Data
+      if: ${{ ! inputs.expected_pass_num }}
+      uses: actions/upload-artifact@v4
+      with:
+        name: Inductor-XPU-E2E-Data-${{ github.event.pull_request.number || github.sha }}
+        path: ${{ github.workspace }}/inductor_log
+
     - name: Upload Inductor XPU E2E Data (${{ inputs.suite }} ${{ inputs.dt }} ${{ inputs.mode }} ${{ inputs.scenario }})
+      if: ${{ inputs.expected_pass_num }}
       uses: actions/upload-artifact@v4
       with:
         name: Inductor-XPU-E2E-Data-${{ inputs.suite }}-${{ inputs.dt }}-${{ inputs.mode }}-${{ inputs.scenario }}-${{ github.event.pull_request.number || github.sha }}
```

Comment:
Comment for the last step. We need consider to create a new file to record the expected pass number for each config, especially for the multi-configs scenario. In the last result check step, if the result check is needed (eg. `inputs.need_check` ), we can read the expected value for each config in the parameters combinations. How do you think?

Reply:
Is it better to check failed number, which should be more easy because it can be 0?

Reply:
But sometimes failed model info didn't written into the CSV due to some reason, so still suggest to add such config. How do you think?

It can be a csv file, e.g.
|suite|dt|mode|scenario|pass_num|
|--|--|--|--|--|
|huggingface|float32|training|accuracy|46|